go.mod

```diff
@@ -96,6 +96,8 @@ require (
 )
 
 replace (
+	golang.org/x/net => golang.org/x/net v0.0.0-20211209124913-491a49abca63 // fixes CVE-2021-44716
```

Comment:
We don't need this. Can you please run this instead, and then drop this:

```
go get -v -d golang.org/x/net@491a49abca63
```

Reply:
As mentioned on Slack, I think it's somewhere down in the dependency tree as it's not in `go.mod` but in `go.sum`, and this overrides it for anyone importing it, but we don't have it in `go.mod` so I wouldn't import it via `go get` to be honest.